### ISSUE TITLE
Option to use sudo for package install action (ubuntu OS)

### ISSUE THREAD

**Issue by muhannad0** [comtrya/comtrya#49]
As a non-root user, when executing a manifest to install packages, comtrya reports that package was installed successfully even though the package was not actually installed.

It would be good to have an option like `require_sudo: true` for package install actions (maybe globally on the manifest or per action).

Machine details:
- OS: Ubuntu 20.04 LTS
- `ubuntu` user able to sudo without password

Test Output:
```bash
ubuntu@ip-192-168-21-173:~/comtrya/dev$ comtrya -vv docker
 TRACE manifest_directory="/home/ubuntu/comtrya/dev/docker" manifests="" Comtrya execution started
 DEBUG OS Detected OS="Ubuntu"
 TRACE Building Contexts
 DEBUG context="user" key="name" value="Ubuntu"
 DEBUG context="user" key="username" value="ubuntu"
 DEBUG context="user" key="home_dir" value="/home/ubuntu"
 DEBUG context="user" key="config_dir" value="/home/ubuntu/.config"
 TRACE manifest_load{manifest="main.yaml"}: manifest="main.yaml"
 TRACE manifest_load{manifest="main.yaml"}: absolute_path="/home/ubuntu/comtrya/dev/docker/main.yaml"
 TRACE manifest_load{manifest="main.yaml"}: template="actions:\n  - action: package.install\n    name: docker\n\n"
 TRACE manifest_load{manifest="main.yaml"}: rendered="actions:\n  - action: package.install\n    name: docker\n\n"
 TRACE manifest_load{manifest="main.yaml"}: Registered Manifest manifest="docker"
 DEBUG manifests=""
  INFO manifest_run{manifest="docker"}:package.install{provider="Aptitude"}: Package Installed packages="docker"
 DEBUG manifest_run{manifest="docker"}: Packages installed successfully
  INFO manifest_run{manifest="docker"}: Completed

ubuntu@ip-192-168-21-173:~/comtrya/dev$ which docker
ubuntu@ip-192-168-21-173:~/comtrya/dev$ docker

Command 'docker' not found, but can be installed with:

sudo snap install docker     # version 19.03.13, or
sudo apt  install docker.io  # version 19.03.8-0ubuntu1.20.04.1

See 'snap info docker' for additional versions
```

**Comment by rawkode** [comment]
Ah. That’s annoying. Sorry about that @muhannad0 

I’ll get that resolved today 👍

**Comment by rawkode** [comment]
Closes by #50
